### ISSUE TITLE
[bug] Getting [WinError 17] when user_home_short points to different drive and package is installed

### ISSUE THREAD

**Issue by sanko-oz** [conan-io/conan#8063]
When setting up _user_home_short_ to point to another drive I get a [WinError 17] during a package install, which tells me that the conanmanifest.txt cannot be moved.  
The current workaround is to force a full rebuild of the package.

Maybe this is the solution -> https://github.com/tsroten/fcache/issues/28#issuecomment-629452552

### Environment Details (include every applicable attribute)
  * Operating System+version: Windows 7
  * Conan version: 1.31
  * Python version: 3.7.1

**Comment by memsharded** [comment]
Hi @sanko-oz 

Can you please post the whole output, it would be great if you can enable CONAN_VERBOSE_TRACEBACK env-var, so we get the full traceback, to know where exactly is failing.

Yes, it seems the ``shutil.move`` would be a good solution, lets identify where it is failing. Thanks for reporting!

**Comment by memsharded** [comment]
Hi @sanko-oz 

We are getting close to next Conan 1.32 release. If you could help pasting the traceback, we could try to submit a bugfix soon to be included in it. Thanks!

**Comment by sanko-oz** [comment]
Hi @memsharded,

sorry for the late reply.  
This is the traceback:
```
test\\dl\\pkg\\d7485fdf28a1cbd0557b4051350655c87ed93f91\\conanmanifest.txt' -> 'D:\\.conan\\4060e5\\1\\conanmanifest.txt'
Traceback (most recent call last):
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\command.py", line 2108, in run
method(args[0][1:])
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\command.py", line 366, in create
info = self._conan.create(args.path, name, version, user, channel,
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\conan_api.py", line 95, in wrapper
return f(api, *args, **kwargs)
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\conan_api.py", line 387, in create
create(self.app, ref, graph_info, remotes, update, build_modes,
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\cmd\create.py", line 79, in create
deps_install(app=app,
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\manager.py", line 76, in deps_install
installer.install(deps_graph, remotes, build_modes, update, keep_build=keep_build,
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\installer.py", line 306, in install
self._build(nodes_by_level, keep_build, root_node, graph_info, remotes, build_mode, update)
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\installer.py", line 409, in _build
self._download(downloads, processed_package_refs)
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\installer.py", line 398, in _download
_download(node)
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\installer.py", line 387, in _download
self._download_pkg(layout, n)
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\installer.py", line 401, in _download_pkg
self._remote_manager.get_package(node.conanfile, node.pref, layout, node.binary_remote,
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\remote_manager.py", line 142, in get_package
self._get_package(layout, pref, remote, output, recorder)
File "c:\users\eisenack\appdata\local\programs\python\python38-32\lib\site-packages\conans\client\remote_manager.py", line 179, in _get_package
os.rename(file_path, os.path.join(package_folder, file_name))
OSError: [WinError 17] Das System kann die Datei nicht auf ein anderes Laufwerk verschieben: 
```

**Comment by memsharded** [comment]
Thanks @sanko-oz !

https://github.com/conan-io/conan/pull/8103 has been merged, it will be released in 1.32 soon.

**Comment by memsharded** [comment]
It has been backported in https://github.com/conan-io/conan/commit/69d3ecaf3729be362dc286ca448bbbc39a2164f2 to patch release 1.31.4, that should be released today or tomorrow.
